src/core/inline-idl-parser.js

```diff
@@ -4,6 +4,7 @@
 import hyperHTML from "hyperhtml";
 import { showInlineError } from "./utils";
 
+const exceptionRegex = /\B"([^"]*)"\B/; // {{ "SomeException" }}
```

Comment:
Seems exactly same as `enumValueRegex`, are we okay with it?

Reply:
Yeah, it's the same... I might just kill it the enum-value one. Using `Foo["value"]` is better... alternative, is that we support both, but that's really messy.

Reply:
Might be better to remove that first, as it seems this shadows enum one...

Reply:
@sidvishnoi, I'm a bit confused as to how `Enum["value"]` becomes just "value" (which is good... but just can't figure it out).

Reply:
Never mind... figured it out 🤪